bin/metrics-mesos.rb

```diff
@@ -33,6 +33,10 @@
 require 'socket'
 require 'json'
 
+# Mesos default ports are defined here: http://mesos.apache.org/documentation/latest/configuration
+MASTER_DEFAULT_PORT = '5050'
+SLAVE_DEFAULT_PORT = '5051'
```

Comment:
I copied what you already had in check-mesos.rb. If the default ports are 5050 for master nodes and 5051 for slave nodes, then you wouldn't be able to support both unless you did it this way.